addons/payment_mollie_official/models/payment_acquirer_method.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+
+import logging
+
+from odoo import models, fields, api, _
+
+_logger = logging.getLogger(__name__)
+
+
+def get_base_url(env):
+    base_url = env['ir.config_parameter'].sudo().get_param('web.base.url')
+    return base_url
+
+
+def get_mollie_provider(env):
+    provider = env['payment.acquirer'].sudo()._get_main_mollie_provider()
+    return provider
+
+
+def get_mollie_provider_key(env):
+    provider = env['payment.acquirer'].sudo()._get_main_mollie_provider()
+    key = provider._get_mollie_api_keys(provider.state)['mollie_api_key']
+    return key
+
+
+class PaymentAcquirerMethod(models.Model):
```

Comment:
why is this needed? I don't see any setting in the payment method section of the mollie backend to change the currency of the method or anything.
Traditionnally in Odoo, payment methods are handled on the acquirer side; e.g. with an integrated checkout. There should be no need to configure this in Odoo